### ISSUE TITLE
Sort the alarms in chronological/alarm time order (autosort)

### ISSUE THREAD

**Issue by JamesDean829** [fennifith/Alarmio#146]
Hi,
currently all alarms are sorted by date of creation.
An option which sorts them by the time they ring whould be very nice!
As someone that uses alarms for everything, this is more of an inconvenience when hunting down a previous alarm to use again.

**Comment by fennifith** [comment]
Yes, that would certainly make sense as the default behavior! I think the "date of creation" sorting is just a side-effect of not currently having any sorting implemented, not an intentional functionality (I wasn't thinking too hard about that when I was writing it).
